README.md

```diff
@@ -610,6 +613,22 @@ Usage: `nerdctl create [OPTIONS] IMAGE [COMMAND] [ARG...]`
 
 The `nerdctl create` command similar to `nerdctl run -d` except the container is never started. You can then use the `nerdctl start <container_id>` command to start the container at any point.
 
+### :whale: nerdctl cp
+Copy files/folders between a running container and the local filesystem
+
+Usage:
+- `nerdctl cp [OPTIONS] CONTAINER:SRC_PATH DEST_PATH|-`
+- `nerdctl cp [OPTIONS] SRC_PATH|- CONTAINER:DEST_PATH`
+
+Flags:
+- :whale: `-L, --follow-link` Always follow symbol link in SRC_PATH.
+-  `--no-preserve` The copied file/directory's ownership and permissions will not be preserved in the container.
```

Comment:
We use 🤓 emoji for marking nerdctl-specific flags

Reply:
Also I'm curious whether we can implement `docker cp --archive` and remove this nerdctl-specific flag

Reply:
two flags are very close. I will convert this in a follow-up PR :)

Reply:
@AkihiroSuda I  added the `archive` flag and I kept `no-preserve` to handle permissions. WDYT ?

Reply:
I'd prefer not to keep `no-preserve` in this PR.
Can be discussed later if it is really useful.

Reply:
fixed